cibuildwheel/macos.py

```diff
@@ -58,14 +58,11 @@ def shell(args, env=None, cwd=None):
             before_build_prepared = prepare_command(before_build, python=python, pip=pip)
             shell(shlex.split(before_build_prepared), env=env)
 
-        # install the package first to take care of dependencies
-        shell([pip, 'install', project_dir], env=env)
-
         # build the wheel to temp dir
         temp_wheel_dir = '/tmp/tmpwheel%s' % config.version
         shell([pip, 'wheel', project_dir, '-w', temp_wheel_dir, '--no-deps'], env=env)
-        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
 
+        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
```

Comment:
Is there a chance that multiple wheels get built?
I don't see how or why one package could do that, but the Linux version is looping over all wheels. So for consistency we can maybe either change this line, or the one in linux.py?